e2e/staticpvc.go

```diff
@@ -159,3 +159,116 @@ func validateRBDStaticPV(f *framework.Framework, appPath string, isBlock bool) e
 	execCommandInPod(f, cmd, rookNamespace, &listOpt)
 	return nil
 }
+
+func validateCephFsStaticPV(f *framework.Framework, appPath, scPath string) error {
+	opt := make(map[string]string)
+	var (
+		cephFsVolName = "staticCephFsVol"
+		groupName     = "e2e"
+		fsName        = "myfs"
+		pvName        = "pv-name"
+		pvcName       = "pvc-name"
+		// minikube creates default class in cluster, we need to set dummy
```

Comment:
default storage class

Reply:
Its dummy only, we are not creating it.

Reply:
```suggestion
              // minikube creates default storage class in cluster, we need to set dummy
```
this is what humble meant